docs/Using-Fleet/REST-API.md

```diff
@@ -1690,21 +1690,21 @@ None.
 | order_key               | string  | query | What to order results by. Can be any column in the hosts table.                                                                                                                                                                                                                                                                             |
 | after                   | string  | query | The value to get results after. This needs order_key defined, as that's the column that would be used.                                                                                                                                                                                                                                      |
 | order_direction         | string  | query | **Requires `order_key`**. The direction of the order given the order key. Options include `asc` and `desc`. Default is `asc`.                                                                                                                                                                                                               |
-| status                  | string  | query | Indicates the status of the hosts to return. Can either be `new`, `online`, `offline`, or `mia`.                                                                                                                                                                                                                                            |
+| status                  | string  | query | Indicates the status of the hosts to return. Can either be `new`, `online`, `offline`, `mia` or `missing`.                                                                                                                                                                                                                                            |
```

Comment:
I don't think it was specified in the ticket, but the `status` parameter is used by a number of endpoints. In addition to `GET /api/v1/fleet/hosts`, you should update the documentation for that parameter for:

* `GET /api/v1/fleet/hosts/count` (count hosts)
* `POST /api/v1/fleet/hosts/transfer/filter` (transfer hosts to a team by filter)
* `POST /api/v1/fleet/hosts/delete` (bulk delete hosts by filter or ids)
* `GET /api/v1/fleet/hosts/report` (get hosts report in CSV)
* `GET /api/v1/fleet/labels/{id}/hosts` (list hosts in a label)

They all end up calling `filterHostsByStatus`, so that's just the documentation that needs adjusting (the implementation should already support it).

Reply:
+1 Now I added that new value to other lists of status-values in this document.